mne_connectivity/spectral/epochs.py

```diff
@@ -1513,10 +1570,13 @@ def spectral_connectivity_epochs(data, names=None, method='coh', indices=None,
         'mim', 'gc', 'gc_tr]``) cannot be called with the other methods.
     indices : tuple of array | None
         Two arrays with indices of connections for which to compute
-        connectivity. If a multivariate method is called, the indices are for a
-        single connection between all seeds and all targets. If None, all
-        connections are computed, unless a Granger causality method is called,
-        in which case an error is raised.
+        connectivity. If a bivariate method is called, each array for the seeds
+        and targets should contain the channel indices for each bivariate
+        connection. If a multivariate method is called, each array for the
+        seeds and targets should consist of nested arrays containing
+        the channel indices for each multivariate connection. If ``None``,
+        connections between all channels are computed, unless a Granger
+        causality method is called, in which case an error is raised.
```

Comment:
Do we need to discuss anything about the masking behavior? Or would we say that's an internal implementation detail?

If it's an internal implementation detail, I wonder if we can perhaps add inline comments where it first comes up about how the `indices` variable changes into a mask.

I'm okay being vetoed here. Just a thought

Reply:
In terms of what is passed into the function, the masking is an internal feature. At present there is only a small inline comment documenting this:
https://github.com/mne-tools/mne-connectivity/blob/883e816e097c1894008ac088733910b8aca87d29/mne_connectivity/spectral/epochs.py#L118-L119
... although the function where the masking happens is documented:
https://github.com/mne-tools/mne-connectivity/blob/883e816e097c1894008ac088733910b8aca87d29/mne_connectivity/utils/utils.py#L86-L144
I'm happy to add more comments directly in `epochs.py` and `time.py`.

Reply:
Of course, as soon as the connectivity object is returned, the masking is no longer just an internal implementation. The goal of this example was to provide some more context on this:
https://github.com/mne-tools/mne-connectivity/blob/883e816e097c1894008ac088733910b8aca87d29/examples/handling_ragged_arrays.py#L1-L8

However I realise adding some documentation within the connectivity classes themselves also makes sense, e.g. here:
https://github.com/mne-tools/mne-connectivity/blob/883e816e097c1894008ac088733910b8aca87d29/mne_connectivity/utils/docs.py#L36-L43
Happy to add this as well.

Reply:
https://github.com/mne-tools/mne-connectivity/pull/142/commits/f5c1c3e52eb0fe78c7287a272dbac4f9f6f9afc7:
- Expanded upon the inline comments per the suggestion.
- Was also able to clean up some code now that `_check_multivariate_indices` is private.